content/en/blog/_posts/2022-11-27-Vulnerable-Microservices/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2022-11-27
+slug: security-behavior
+---
+
+
+**Author:**
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy that relies on constructing a non-vulnerable, weakness-free service in 2022 is clearly too naïve. It seems the only viable strategy is to:
+
+***
+
+<h3 align="center">
+Admit that your services are vulnerable
+</h3>
+
+***
```

Comment:
```suggestion
**Admit that your services are vulnerable**
```

We'll only accept blog articles that follow our style conventions. If that's a deal-breaker, we can discuss this - but I don't think the blog team are likely to budge.

Reply:
ok, let me try another approach - I am trying to ensure this sentence properly stick-out, I will send s revised option, hopefully, one that can be accepted.